bentoml/server/api_server.py

```diff
@@ -454,6 +454,7 @@ def handle_ml_processing(self):
         logger.info('requests status_code for GET subscriptionQuery: %s', r.status_code)
         logger.info('Data: %s', r.json())
         ENTITY_INPUT_DATA = r.json()['entityID']['value']
```

Comment:
I did not really pay attention to this the first time, but it does not match the definition of a SubscriptionQuery entity.

Reply:
What is it then?

Reply:
The answer is in the specification :)

Reply:
What's the difference causes problem ?

{
  "id":"urn:ngsi-ld:SubscriptionQuery:01",
  "type":"SubscriptionQuery",
  "entities": [
    {
      "type": "WaterConsumption"
    }
  ],
  “q”: “refCity==urn:ngsi-ld:City:Valbonne” 
}


{
    'id': 'urn:ngsi-ld:MLProcessing:SubscriptionQuery:e7be459e-dcee-46ab-90da-fba3120db4ff',
    'type': 'https://uri.fiware.org/ns/data-models#SubscriptionQuery',
    'entityID': {
        'type': 'Property',
        'value': 'urn:ngsi-ld:River:014f5730-72ab-4554-a106-afbe5d4d9d26'
    },
    'query': {
        'type': 'Property',
        'value': '?attrs=precipitation'
    },
    '@context': ['https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld']
}

Reply:
The thing with the `SubscriptionQuery` in the specification is that it is the same syntax as a regular `Subscription`, so the user does not have to learn a new syntax to subscribe to something. But let's keep this for a later iteration.

Reply:
I don't understand. It's better to talk.